Plugins/BackbonePluginTemplatePlugin/BackbonePluginTemplatePluginGenerator.cs

```diff
@@ -131,6 +140,87 @@ private GlobalJsonModification CreateMsBuildSdk()
             return globalJsonModificationBuilder.Build();
         }
 
+        private Task AddProjectReferences()
+        {
+            var packagesReferences = CreateProjectModification();
+            return _projectManipulator.ManipulateProject(GetProjectFilePath(), packagesReferences);
+        }
+
+        private Task AddTestProjectReferences()
+        {
+            var testProjectModification = CreateTestProjectModification();
+            return _projectManipulator.ManipulateProject(GetTestProjectFilePath(), testProjectModification);
+        }
+
+        private ProjectModification CreateProjectModification()
+        {
+            var projectModificationBuilder = new ProjectModificationBuilder();
+            if (PluginVariant != VariantType.MinimalPlugin)
+            {
+                projectModificationBuilder = projectModificationBuilder.AddPackageReferences(CreateExtendedProjectPackageReferences());
+            }
+
+            if (UsesCentralPackageVersionsSdk)
+            {
+                projectModificationBuilder = projectModificationBuilder.AddSdk(CentralPackageVersions);
+            }
+
+            return projectModificationBuilder.Build();
+        }
+
+        private ProjectModification CreateTestProjectModification()
+        {
+            var projectModificationBuilder = new ProjectModificationBuilder();
+            projectModificationBuilder = projectModificationBuilder.AddPackageReferences(CreateTestProjectPackageReferences());
```

Comment:
Make these two assignments one:
```c#
new ProjectModificationBuilder()
    .AddPackageReferences(CreateTestProjectPackageReferences())
```

Reply:
Addressed in: [Refactor ProjectModifictionBuilder call](https://github.com/messerli-informatik-ag/meta-generator/pull/27/commits/2d3e04912ceb9f49fe882885263959f10e13a8c0)